src/components/MetricChart/MetricChart.js

```diff
@@ -119,6 +119,29 @@ const GenericMetricChart = ({ chartConfig, showGrid }) => {
       })
     }
 
+    if (chartConfig.gradient) {
+      const canvasHeight = showGrid ? 200 : 80
+      if (chartInstance.current.options.scales.x.grid.display !== showGrid) {
+        chartInstance.current.options.scales.x.grid.display = showGrid
+        chartInstance.current.options.scales.y.grid.display = showGrid
+
```

Comment:
you added empty lines randomly. You don't need it here and you haven't added it before `if`

Reply:
done